apps/server/src/modules/tool/external-tool/uc/external-tool.uc.spec.ts

```diff
@@ -380,6 +443,88 @@ describe('ExternalToolUc', () => {
 
 			expect(result).toEqual(updatedExternalToolDO);
 		});
+
+		describe('when tool has no logo url', () => {
+			const setup2 = () => {
```

Comment:
setup2 sounds bad. rename to something more fitting, maybe "setupLogo" like further down?

Reply:
Was intended.. but i changed it